scripts/testApp.sh

```diff
@@ -37,33 +40,34 @@ kubectl get all -n istio-system
 
 SYSTEM=$(kubectl get pods | grep system | sed 's/ .*//')
 
-kubectl exec -it $SYSTEM -- /opt/ol/wlp/bin/server pause
+kubectl exec -it "$SYSTEM" -- /opt/ol/wlp/bin/server pause
 
 sleep 60
 
-export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
+export INGRESS_PORT
+INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
 
-echo $(minikube ip):$INGRESS_PORT
+echo "$(minikube ip)":"$INGRESS_PORT"
 
-curl -H Host:inventory.example.com http://$(minikube ip):$INGRESS_PORT/inventory/systems/system-service -I
+curl -H Host:inventory.example.com http://"$(minikube ip)":"$INGRESS_PORT"/inventory/systems/system-service -I
 
-if [ $? -ne 0 ]; then
+if mycmd; then
```

Comment:
`mycmd; ` is not right and I found the test log as
![Screen Shot 2021-05-17 at 11 28 58 AM](https://user-images.githubusercontent.com/29576299/118515220-33fdfb80-b703-11eb-838e-bedacfdca834.png)